Streamlink Twitch GUI/tools/chocolateyinstall.ps1

```diff
@@ -4,6 +4,13 @@ $toolsDir = Split-Path -parent $MyInvocation.MyCommand.Definition
 $installDir = Split-Path -parent $toolsDir
 $downloadPath = "https://github.com/streamlink/streamlink-twitch-gui/releases/download/$packageVersion/"
 
+New-Item -ItemType file $installDir\$packageName\notification_helper.exe.ignore -EA 0 | Out-Null

```

Comment:
I've tried this out locally and I'm still seeing `notification_helper.exe` get shim'd into `C:\ProgramData\chocolatey\bin`. I assume this should be fixed as a part of this PR? (Or maybe there's something I'm missing)

Reply:
What the hell, I literally included it. Looks like [`3e9885d` (#41)](https://github.com/streamlink/streamlink-twitch-gui-chocolatey/pull/41/commits/3e9885d8859b7dd5c8d412003e1eb9e1a5c958b6) pummeled it somehow.

Reply:
Resolved by [`318cded` (#41)](https://github.com/streamlink/streamlink-twitch-gui-chocolatey/pull/41/commits/318cdeddaf0cb301013d4f247df8b20956dc30de)